code/modules/hydroponics/plant_genes.dm

```diff
@@ -5,15 +5,23 @@
 	/// The font awesome icon name representing the gene in the seed extractor UI
 	var/icon = "dna"
 	/// Flags that determine if a gene can be modified.
-	var/mutability_flags
-
+	var/mutability_flags =  PLANT_GENE_MUTATABLE | PLANT_GENE_GRAFTABLE
 /*
  * Returns the formatted name of the plant gene.
  *
  * Overridden by the various subtypes of plant genes to format their respective names.
  */
 /datum/plant_gene/proc/get_name()
```

Comment:
For these procs, make an override in a new file, don't supercall (. = ..()), and slap this proc in
and revert this file to original

you can re declare the vars there too